salt/config/__init__.py

```diff
@@ -43,9 +43,10 @@
 
 try:
     import psutil
-    psutil.virtual_memory
+    if not hasattr(psutil, 'virtual_memory'):
+        raise ImportError('Version of psutil too old.')
```

Comment:
Great article. I didn't know that. Thanks for the info here, @srkunze . I learned something new. :]

@guettli Apologies for my ignorance on this one.